crates/wit-component/tests/components.rs

```diff
@@ -153,12 +154,25 @@ fn run_test(path: &Path) -> Result<()> {
         }
     };
 
+    let features =
+        WasmFeatures::WASM2 | WasmFeatures::COMPONENT_MODEL | WasmFeatures::COMPONENT_MODEL_ASYNC;
+    Validator::new_with_features(features)
```

Comment:
Similar to a few other locations I think it's fine to use `WasmFeatures::all()` here